tests/topotests/bgp_srv6l3vpn_sid/r1/zebra.conf

```diff
@@ -0,0 +1,43 @@
+log file zebra.log
+!
+hostname r1
+password zebra
+!
+log stdout notifications
+log monitor notifications
+log commands
+!
+debug zebra packet
```

Comment:
Please comment debug statements. We do not allow `debug` for CI due to disk space issues, etc.

Reply:
Done